core/tasks.ts

```diff
@@ -49,6 +55,10 @@ export class Task {
     return this;
   }
 
+  public getStatement() {
+    return this.proto.statement;
+  }
```

Comment:
Once tasks are assembled you can't read the statement. I think you should be able to, so I added this, which made modifying `Tasks` post adapter assembly easier.